mwviews/page_view_count.py

```diff
@@ -12,7 +12,18 @@ def __init__(self, project, page_name, views, bytes_returned):
 
     @classmethod
     def from_line(cls, line):
-        project, page_name, views, bytes_returned = line.strip().split(" ")
+        line_split_on_spaces = line.strip().split(" ")
```

Comment:
I don't see why this is necessary.  Spaces don't exist inside of titles in the file format.  It seems like this code only exists to address a problem we caused by producing output data that had spaces in titles.  I think we should fix that problem separately and allow for simple line parsing here.